tools/scripts/clang-format.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# Copyright (c) 2020, Erik Henriksson
+#
+# This file is part of the modm project.
+#
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#----------------------------------------------------------------------------
+
+git diff --name-only --diff-filter=A -C -M develop | grep -e "\.\(c\|h\|hpp\|cpp\)\$" | xargs clang-format -i "$@"
```

Comment:
Added a filter so that only newly added files get formatted, otherwise I go crazy as discussed ;-P

Reply:
Thanks! I realized this command is even better, as it reformats only changed lines: `git clang-format upstream/develop`